File: qqbot-module-platform-admin/src/main/java/com/mieai/qqbot/admin/onboarding/OnboardingStatusResponse.java
package com.mieai.qqbot.admin.onboarding;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.mieai.qqbot.admin.database.DatabaseType;
import java.time.Instant;

@JsonInclude(JsonInclude.Include.ALWAYS)
public record OnboardingStatusResponse(
        OnboardingStage stage,
        DatabaseType databaseType,
        long botCount,
        Instant completedAt) {
}

File: qqbot-module-platform-admin/src/main/kotlin/com/mieai/qqbot/admin/onboarding/OnboardingStatusResponse.kt
package com.mieai.qqbot.admin.onboarding

import com.fasterxml.jackson.annotation.JsonInclude
import com.mieai.qqbot.admin.database.DatabaseType
import java.time.Instant

@JsonInclude(JsonInclude.Include.ALWAYS)
data class OnboardingStatusResponse(
    val stage: OnboardingStage,
    val databaseType: DatabaseType?,
    val botCount: Long,
    val completedAt: Instant?,
)
